backend/src/main/resources/db-scripts/migration/V1.8__allow_multiple_attachments_on_handover.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE IF NOT EXISTS handover_mail_attachments (
```

Comment:
Question of interest: When do we keep the attachment in the database? Is it only when an attachment was successfully sent?

Reply:
Yes, as soon as we sent the mail